pkgs/top-level/all-packages.nix

```diff
@@ -13738,6 +13738,8 @@ with pkgs;
 
   hexedit = callPackage ../applications/editors/hexedit { };
 
+  hiawatha = callPackage ../servers/http/hiawatha { };
```

Comment:
Could you move this to the "### SERVERS" section instead, please: 
https://github.com/ndowens/nixpkgs/blob/eb232b63ee6b6bb5d2b437694afa0ad1f2617c33/pkgs/top-level/all-packages.nix#L10239
After hbase:
https://github.com/ndowens/nixpkgs/blob/eb232b63ee6b6bb5d2b437694afa0ad1f2617c33/pkgs/top-level/all-packages.nix#L10410
Thanks!